--- a/freerdp.spec.orig
+++ b/freerdp.spec
@@ -2,7 +2,7 @@
 # "--with=ffmpeg" to mock/rpmbuild; or by globally
 # setting these variables:
 # https://bugzilla.redhat.com/show_bug.cgi?id=2242028
-#global _with_ffmpeg 1
+%global _with_ffmpeg 1
 
 # Can be rebuilt with OpenCL support enabled by passing # "--with=opencl"
 # or by globally setting:
@@ -28,9 +28,9 @@
 #global _with_webview 1
 
 Name:           freerdp
-Epoch:          2
+Epoch:          3
 Version:        3.12.0
-Release:        1%{?dist}
+Release:        1.ts1%{?dist}
 Summary:        Free implementation of the Remote Desktop Protocol (RDP)
 
 # The effective license is Apache-2.0 but:
@@ -238,6 +238,10 @@ find . -name "*.c" -exec chmod 664 {} \;
     -DWITH_XTEST=%{?_with_server:ON}%{?!_with_server:OFF} \
     -DWITH_XV=ON \
     -DWITH_ZLIB=ON \
+    -DWITH_INTERNAL_RC4=ON \
+    -DWITH_INTERNAL_MD4=ON \
+    -DWITH_INTERNAL_MD5=ON \
+    -DCHANNEL_RDPECAM_CLIENT=ON \
 %ifarch x86_64
     -DWITH_SSE2=ON \
     -DWITH_VAAPI=%{?_with_ffmpeg:ON}%{?!_with_ffmpeg:OFF} \
